build/package.logging.nlog.nuspec

```diff
@@ -15,7 +15,7 @@
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
     <tags>rabbitmq management api http chinchilla</tags>
     <dependencies>
-      <dependency id="NLog" version="4.2.0" />
+      <dependency id="NLog" version="5.0.0-beta10" />
```

Comment:
This makes me nervous at the moment. No fully fledged support for netstandard at this point, so had to rely on the pre-release. Worth waiting before merging this?

Reply:
I think this is ok for now. Will be a netstandard one soon I think and we don't need to take this dep.